src/cpp/common/auth_property_iterator.cc

```diff
@@ -61,7 +61,7 @@ bool AuthPropertyIterator::operator!=(const AuthPropertyIterator& rhs) const {
   return !operator==(rhs);
 }
 
-const AuthProperty AuthPropertyIterator::operator*() {
+AuthProperty AuthPropertyIterator::operator*() {
   return std::pair<grpc::string_ref, grpc::string_ref>(
```

Comment:
Interesting design! Ignore my earlier comment, return by value was intended here, so the const is superfluous. string_refs appear to be immutable by design as well.

Reply:
Yeah, you're right.